build/README.md

```diff
@@ -171,12 +171,12 @@ a PR to that effect would be greatly appreciated.
 1.  Use [`make-certs.py` script](./make-certs.py) to create certificates for
     the new CockroachDB cluster:
 
-        ./make-certs.py --cluster $CLUSTER_CONTEXT --namespace $NAMESPACE
+        python3 ./make-certs.py --cluster $CLUSTCONTEXT --namespace $NAMESPACE
```

Comment:
I think we want to keep `$CLUSTER_CONTEXT` rather than replacing with `$CLUSTCONTEXT`

Reply:
Oops, it is a typo. Thank you for pointing out.